### ISSUE TITLE
[Bug][a11y]Blue selection rectangle improperly placed on input text boxes from Sync 

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#5082]
## Prerequisites
Have Voice Assistant enabled 

## Steps to reproduce
1. Tap on the upper right 3dot button
2. Tap Settings
3. Tap "Turn on Sync"
4. Tap on the email input box

### Expected behavior
Blue selection rectangle is displayed and focused on the email input text box

### Actual behavior
After selecting the email input text box and starting to type the rectangle is missplaced 

### Device information

* Android device: Samsung Galaxy Tab A (Android 9)
* Fenix version: 
 • 2.0.0-rc.1
• Nightly Build  #12460618 from 3/9

### Notes
► Video
![20190903_125501](https://user-images.githubusercontent.com/51314259/64165898-a014cf00-ce4e-11e9-89b3-73ae895cd099.gif)

**Comment by mcarare** [comment]
For QA: Please check of issue is still reproducible! TY!

**Comment by AndiAJ** [comment]
Hi, this is **still reproducible** on the latest **Nightly Build #13440604 from 12/10** using a Samsung Galaxy Tab A (Android 9)

► Video
![20191210_100139](https://user-images.githubusercontent.com/51314259/70507311-c07eb480-1b34-11ea-94ba-1308754c728d.gif)

**Comment by vesta0** [comment]
@asadotzler and @eeejay  please let us know if this is a priority from an accessibility perspective.

**Comment by eeejay** [comment]
This is not the highest priority, and should be considered polish.
